plugins/pluto/.gitignore

```diff
@@ -0,0 +1,2 @@
+include/plutoplugin/scopy-pluto_export.h
```

Comment:
this should be include/pluto not plutoplugin similar with what's been mentioned in previous comments about all the other channels